docker-compose.yml

```diff
@@ -143,7 +143,7 @@ services:
         bundle exec rake db:setup;
       fi;
       RAILS_ENV=test bundle exec rake db:create;
-      bundle exec rake db:test:prepare;
+      RAILS_ENV=test bundle exec rake db:test:prepare;
```

Comment:
Can we better define a env variable `RAILS_ENV`set to `test` in this pod ? I see this variable set for multiple commands already

Reply:
I don't like that idea, the purpose of this container is to set up the development environment and on top of that also sets up the testing one. I would rather keep the latter as explicit

Reply:
In that case we would have to set the `RAILS_ENV=development` on the rest of the commands, even in the `if` statement above and the word itself is longer so I think this is simpler :laughing:

Reply:
fine by me :+1:  

but I think that'd be explicit enough , the template would look like similar to:

```YAML
  build-rails-db:
    image: compliance-backend-rails
    environment:
        - RAILS_ENV=test
    command: sh -c '
      if bundle exec rake db:version 2> /dev/null; then
        bundle exec rake db:migrate;
      else
        bundle exec rake db:setup;
      fi;
      bundle exec rake db:create;
      bundle exec rake db:test:prepare;
      bundle exec rails db < db/cyndi_setup_devel.sql;
      bundle exec rails db < db/cyndi_setup_test.sql;
      '
```
Though it's true it would affect the three other rake commands `db:version`, `db:migrate` and `db:setup` which - I don't know if that's okay either.

Reply:
Yeah, those would need the development env...